docker/dist/Dockerfile.dist.centos7

```diff
@@ -37,11 +37,20 @@ WORKDIR /heron
 # run heron installer
 RUN /heron/heron-install.sh
 
-RUN mkdir -p /opt/heron && \
-    tar --strip-components=1 -m -zxvf /heron/heron-core.tar.gz -C /heron
+RUN ln -s /usr/local/heron/dist/heron-core /heron
+
+RUN mkdir -p /heron/heron-tools
+RUN ln -s /usr/local/heron/bin /heron/heron-tools
+RUN ln -s /usr/local/heron/conf /heron/heron-tools
+RUN ln -s /usr/local/heron/dist /heron/heron-tools
+RUN ln -s /usr/local/heron/lib /heron/heron-tools
+RUN ln -s /usr/local/heron/release.yaml /heron/heron-tools
+
+RUN ln -s /usr/local/heron/examples /heron
+
+RUN ln -s /usr/local/heron/release.yaml /heron
```

Comment:
Multiple RUN commands increase the number of layers which increase the docker size - you need to optimize to get them into one single RUN command